src/components/coins-page-table/CoinTable/CoinTable.js

```diff
@@ -21,92 +22,98 @@ import {
+  }, [activeDirection]);
 
   useEffect(() => {
-    props.getCoinsDataByCategory(activeCategory)
-  }, [activePageNum])
+    props.getCoinsDataByCategory(activeCategory);
+  }, [activePageNum]);
 
   useEffect(() => {
-    props.getCoinsDataByCategory(activeCategory)
-  }, [activeResultsPerPage])
+    props.getCoinsDataByCategory(activeCategory);
+  }, [activeResultsPerPage]);
 
   return (
-    <OutsideWrapper>
+    <OutsideWrapper width={screenWidth}>
       <TableFilters />
-      <Wrapper>
-        <HeaderRow>
-          <tr>
-            <THNum>
-              <CoinTableTitle>#</CoinTableTitle>
-            </THNum>
-            <THName>
-              <CoinTableTitle>Name</CoinTableTitle>
-            </THName>
-            <THPrice>
-              <CoinTableTitle>Price</CoinTableTitle>
-            </THPrice>
-            <TH1h>
-              <CoinTableTitle>1h%</CoinTableTitle>
-            </TH1h>
-            <TH24h>
-              <CoinTableTitle>24h%</CoinTableTitle>
-            </TH24h>
-            <TH7d>
-              <CoinTableTitle>7d%</CoinTableTitle>
-            </TH7d>
-            <TH24VolMarCap>
-              <CoinTableTitle>24h Volume/Market Cap</CoinTableTitle>
-            </TH24VolMarCap>
-            <THCircTotSup>
-              <CoinTableTitle>Circulating/Total Supply</CoinTableTitle>
-            </THCircTotSup>
-            <THLast7d>
-              <CoinTableTitle>Last 7d</CoinTableTitle>
-            </THLast7d>
-          </tr>
-        </HeaderRow>
-        {!isLoading ? props.coins.map((coin) => (
-          <TableRow key={coin.id} coin={coin} />
-        )) : loaders.map((index) => <LoadingTableRow key={index} />)}
-      </Wrapper>
+      <ScrollWrapper>
+        <Wrapper>
+          <HeaderRow>
+            <tr>
+              <THNum>
+                <CoinTableTitle>#</CoinTableTitle>
+              </THNum>
+              <THName>
+                <CoinTableTitle>Name</CoinTableTitle>
+              </THName>
+              <THPrice>
+                <CoinTableTitle>Price</CoinTableTitle>
+              </THPrice>
+              <TH1h>
+                <CoinTableTitle>1h%</CoinTableTitle>
+              </TH1h>
+              <TH24h>
+                <CoinTableTitle>24h%</CoinTableTitle>
+              </TH24h>
+              <TH7d>
+                <CoinTableTitle>7d%</CoinTableTitle>
+              </TH7d>
+              <TH24VolMarCap>
+                <CoinTableTitle>24h Volume/Market Cap</CoinTableTitle>
+              </TH24VolMarCap>
+              <THCircTotSup>
+                <CoinTableTitle>Circulating/Total Supply</CoinTableTitle>
+              </THCircTotSup>
+              <THLast7d>
+                <CoinTableTitle>Last 7d</CoinTableTitle>
+              </THLast7d>
+            </tr>
+          </HeaderRow>
+          {!isLoading
+            ? props.coins.map((coin) => <TableRow key={coin.id} coin={coin} />)
+            : loaders.map((index) => <LoadingTableRow key={index} />)}
```

Comment:
can you make a component TableLoader  that handles all this stuff: `loaders.map((index) => <LoadingTableRow key={index} />)` and `const loaders = Array.apply(null, Array(20)).map(function () {});`

Seems unnecessary to have it here in coin table

Reply:
I'll fix this in a future commit.